steps/swiftlint-extended/1.0.0/step.yml

```diff
@@ -0,0 +1,106 @@
+title: Swiftlint
+summary: |
+  Generates swiftlint reports in different formats, compatible with Bitrise test reports add-on
+description: |
+  This step generates the swiftlint report and saves it to a file in the bitrise deploy folder. It produces 2 environment variables, one with the report and the other one with the path to the report file. The first variable is useful if you want to p pass your report directly to somewhere, for example a Slack channel. The second variable is useful if you want to process the report file further, for example upload it to S3 or attach it to a pull request. Finally, this step is compatible with the Bitrise test reports add-on, so in case the linting fails, you can easily see the details there.
```

Comment:
NOTE: `this step is compatible with the Bitrise test reports add-on` might worth adding which `reporter` needs to be selected for the Testing add-on support. I would also highlight this information in the input description.

Reply:
@godrei Regardless of the reporter selection, it will work. If the user selects something else than `junit` then another lint run is done to generate the `junit` report and always support the test reports add-on.